contracts/SupplyChain.sol

```diff
@@ -1,45 +1,52 @@
 pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
 
-import "./Package.sol";
-import "./ProducersManager.sol";
+import "./DrugItem.sol";
+import "./VendorsManager.sol";
 
 
-contract SupplyChain is ProducersManager {
+contract SupplyChain is VendorsManager {
 
-    mapping(bytes => Package) private packages;
+    mapping(bytes => DrugItem) private items;
```

Comment:
Under the hood, it's not mapping from bytes => DrugItem. It's bytes32 => DrugItem. Make it explicit.